server/internal/infrastructure/mongo/mongodoc/item.go

```diff
@@ -0,0 +1,102 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/item"
+	"github.com/reearth/reearth-cms/server/pkg/schema"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type ItemDocument struct {
+	ID     string
+	Schema string
+	Fields []ItemFieldDoc
+}
+type ItemFieldDoc struct {
+	SchemaField string
+	ValueType   string
+	Value       any
+}
+
+type ItemConsumer struct {
+	Rows item.List
+}
+
+func (c *ItemConsumer) Consume(raw bson.Raw) error {
+	if raw == nil {
+		return nil
+	}
+
+	var doc ItemDocument
+	if err := bson.Unmarshal(raw, &doc); err != nil {
+		return err
+	}
+	item, err := doc.Model()
+	if err != nil {
+		return err
+	}
+	c.Rows = append(c.Rows, item)
+	return nil
+}
```

Comment:
See https://github.com/reearth/reearth-cms/blob/main/server/internal/infrastructure/mongo/mongodoc/model.go#L63

```suggestion
type ItemConsumer = mongox.SliceFuncConsumer[*ItemDocument, *item.Item]

func NewItemConsumer() *ItemConsumer {
	return NewComsumer[*ItemDocument, *item.Item]()
}
```